src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.Budget;
+import seedu.duke.data.ShoppingList;
+
+public class DeleteCommand extends Command {
+
+    private static int index;
+
+    public static final String COMMAND_WORD = "DEL";
+
+    public static final String DELETE_MESSAGE = "Got it! I have removed the following item:\n";
+
+    public DeleteCommand(int index) {
+        super();
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(ShoppingList items, Budget myBudget) {
```

Comment:
Also if we always have to parse in the budget var, functions like these will never actually use the variable itself? 😨  unless it's a command having to deal with the budget